doc/specs/#1620 - Installer selector on multiple matches.md

```diff
@@ -0,0 +1,93 @@
+---
+author: Darshan Rander @SirusCodes
+created on: 2022-10-29
+last updated: 2022-10-29
+issue id: 1620
+---
+
+# Spec Title
+
+For [#1620](https://github.com/microsoft/winget-cli/issues/1620)"
+
+## Abstract
+
+The winget.exe client simply lists all the matches and if the user wants to install the app they need to re-run the command with Id.
+
+The UX can be improved here by giving users an option to select the package instead of asking them to re-run the command with Id.
+
+## Inspiration
+
+It is not a good experience to copy Id and then run the command again.
+
+## Solution Design
+
+If we find multiple packages then the user should get a selector.
+
+winget should ask for user input, for example -
+
+```pwsh
+> winget install Powershell
+Multiple packages found matching.
+No. Name       Id                   Source
+-------------------------------------------
+ 1  PowerShell 9MZ1SNWT0N5D         msstore
+ 2  PowerShell Microsoft.PowerShell winget
+
+Please select the packages (eg. 2): <user-input>
+
+Installing <user-input>...
+```
+
+> `<user-input>` - is the input by user
+
+After the user will give the input the installation should start.
+
+## UI/UX Design
+
+The idea is explained in [Solution Design](#solution-design).
+
+## Capabilities
+
+### Accessibility
+
```

Comment:
New text in the menu (e.g `Please select the packages`) will need to be localized. There may need to be considerations made for if the indices used for selecting packages also need to be localized

Reply:
Do windows localize numerical forms automatically?

Reply:
I'm not sure, but it would need to be a consideration that needs to be made as to whether they will or won't be

Reply:
I will verify it and add in the spec